apps/src/code-studio/components/header/SignInCallout.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import cookies from 'js-cookie';
+
+const CALLOUT_COLOR = '#454545';
+const TRIANGLE_BASE = 30;
+const TRIANGLE_HEIGHT = 15;
+const CALLOUT_Z_INDEX = 1040;
+const CALLOUT_TOP = 30;
+
+const HideSignInCallout = 'hide_signin_callout';
+
+const styles = {
+  container: {
+    // The outermost div is relatively positioned so it can be used as a positional
+    // anchor for its children (which will be absolutely positioned to avoid affecting
+    // layout).  This element must be 0-sized to avoid affecting layout.
+    position: 'relative',
+    height: 0,
+    width: 0
+  },
+  content: {
+    position: 'absolute',
+    top: CALLOUT_TOP,
+    right: -90,
+    zIndex: CALLOUT_Z_INDEX,
+    backgroundColor: CALLOUT_COLOR,
+    borderRadius: 3
+  },
+  modalBackdrop: {
+    // Most backdrop attributes come from the 'modal-backdrop' class defined by bootstrap
+    // but we need to override the opacity as the default opacity of 0.8 is too dark.
+    // Note that bootstrap defaults the z-index of the backdrop to 1040.
+    opacity: 0.5
+  },
+  upTriangle: {
+    position: 'absolute',
+    top: CALLOUT_TOP - TRIANGLE_HEIGHT,
+    left: -(TRIANGLE_HEIGHT / 2.0),
+    width: 0,
+    height: 0,
+    borderStyle: 'solid',
+    borderTopWidth: 0,
+    borderRightWidth: TRIANGLE_BASE,
+    borderBottomWidth: TRIANGLE_HEIGHT,
+    borderLeftWidth: TRIANGLE_BASE,
+    borderTopColor: 'transparent',
+    borderRightColor: 'transparent',
+    borderBottomColor: CALLOUT_COLOR,
+    borderLeftColor: 'transparent',
+    zIndex: CALLOUT_Z_INDEX
+  },
+  contentContainer: {
+    display: 'flex',
+    flexDirection: 'row',
```

Comment:
I believe `row` is the default flex-direction, so this shouldn't be necessary to specify explicitly

Reply:
Resolved- removed this flexDirection line